.travis.yml

```diff
@@ -41,24 +41,37 @@ jobs:
     - &STANDARD_TEST_JOB
       stage: Test
       php: 7.2
+      env: DRIVER="xdebug"
       install:
         - if [ "$DEPS" == "LOW" ]; then composer update --prefer-lowest $COMPOSER_FLAGS; else composer install $COMPOSER_FLAGS; fi
+      before_script:
+          - |
+              if [[ "$DRIVER" == "pcov" ]]; then
+                echo > $HOME/.phpenv/versions/$TRAVIS_PHP_VERSION/etc/conf.d/xdebug.ini
+                git clone --single-branch --branch=v1.0.6 --depth=1 https://github.com/krakjoe/pcov
+                cd pcov
+                phpize
+                ./configure
+                make clean install
+                echo "extension=pcov.so" > $HOME/.phpenv/versions/$TRAVIS_PHP_VERSION/etc/conf.d/pcov.ini
+                cd $TRAVIS_BUILD_DIR
+              fi
       script:
-        - if [[ $PHPDBG != 1 ]]; then xdebug-enable; fi
-        - if [[ $PHPDBG != 1 ]]; then $PHPUNIT_BIN; else phpdbg -qrr $PHPUNIT_BIN; fi
+        - if [[ "$DRIVER" == "xdebug" || "$DRIVER" == "none" ]]; then xdebug-enable; fi
```

Comment:
So the point of `DRIVER=none` is to run Infection without any kind of coverage collector, with the coverage data we had from the initial PHPUnit run, am I right?

Reply:
Exactly!